yb-voyager/cmd/analyzeSchema.go

```diff
@@ -217,14 +216,16 @@ const (
 	STORAGE_PARAMETERS_DDL_STMT_ISSUE    = "Storage parameters are not supported yet."
 	ALTER_TABLE_SET_ATTRUBUTE_ISSUE      = "ALTER TABLE .. ALTER COLUMN .. SET ( attribute = value )	 not supported yet"
 	FOREIGN_TABLE_ISSUE_REASON           = "Foreign tables requires manual intervention."
+	DEFERRABLE_CONSTRAINT_ISSUE          = "DEFERRABLE constraints not supported yet"
+	UNSUPPORTED_DATATYPE                 = "Unsupported datatype"
 	UNSUPPORTED_PG_SYNTAX                = "Unsupported PG syntax"
 
 	GIST_INDEX_ISSUE_REASON = "Schema contains GIST index which is not supported."
 	GIN_INDEX_DETAILS       = "There are some GIN indexes present in the schema, but GIN indexes are partially supported in YugabyteDB as mentioned in (https://github.com/yugabyte/yugabyte-db/issues/7850) so take a look and modify them if not supported."
 )
 
 // Reports one case in JSON
-func reportCase(filePath string, reason string, ghIssue string, suggestion string, objType string, objName string, sqlStmt string) {
+func reportCase(filePath string, reason string, ghIssue string, suggestion string, objType string, objName string, sqlStmt string, isDatatypeIssue bool) {
```

Comment:
nit: would be cleaner to just take in the issueType as argument. 
Will be easier to read as well: (true/false is not immediately clear, but specifying UNSUPPORTED_DATATYPE is very clear)